webpack.config.js

```diff
@@ -66,46 +75,42 @@ module.exports = function (env, argv) {
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/overview/index.html'),
         template: 'src/overview/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['overview']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/java-runtime/index.html'),
         template: 'src/java-runtime/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['java-runtime']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/getting-started/index.html'),
         template: 'src/getting-started/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['getting-started']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/ext-guide/index.html'),
         template: 'src/ext-guide/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['ext-guide']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/welcome/index.html'),
         template: 'src/welcome/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['welcome']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/classpath/index.html'),
         template: 'src/classpath/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['classpath']
       }),
       new HtmlWebpackPlugin({
         filename: path.resolve(__dirname, 'out/assets/formatter-settings/index.html'),
         template: 'src/formatter-settings/assets/index.html',
-        inlineSource: '.(js|css)$',
         chunks: ['formatter-settings']
       }),
-      new HtmlWebpackInlineSourcePlugin(),
+      new InlineChunkHtmlPlugin(HtmlWebpackPlugin, [/\.(js)$/]),
```

Comment:
Here we use react-dev-utils/InlineChunkHtmlPlugin for an alternative one since `html-webpack-inline-source-plugin` doesn't support webpack 5. 

Refer to https://github.com/facebook/create-react-app/tree/main/packages/react-dev-utils#new-inlinechunkhtmlpluginhtmlwebpackplugin-htmlwebpackplugin-tests-regex, it works well with HtmlWebpackPlugin 4.x, so we use `4.5.0` of `html-webpack-plugin` and it's corresponding webpack@5. Not sure the version can resolves these CVEs or not.